### PR TITLE
{bio}[intel/2019b] Structure v2.3.4

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3122.skitty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/dbd4987a1742ef7334e283cf71e0614d for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2671.swalot.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/1ba35c93f77225f39b0cc2f02ebe2b59 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2092.delcatty.os - Linux centos linux 7.6.1810, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/4240b25ad7316158fa2a4b2322776fa1 for a full test report.

File: easybuild/easyconfigs/s/Structure/Structure-2.3.4-intel-2019b.eb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+easyblock = 'MakeCp'
+
+name = 'Structure'
+version = '2.3.4'
+
+homepage = 'https://web.stanford.edu/group/pritchardlab/structure.html'
+description = """The program structure is a free software package for using multi-locus genotype data to investigate
+ population structure."""
+
+toolchain = {'name': 'intel', 'version': '2019b'}
```
Comment (Micket) [inline easybuild/easyconfigs/s/Structure/Structure-2.3.4-intel-2019b.eb]: There seems to be no math/mpi used here, so iccifort should suffice

**Comment by Micket** [comment]
Going in, thanks @boegel!

**Pull request merged**